src/frontend/about/components/ContributorsDialog.js

```diff
@@ -24,7 +24,7 @@ const contributors = [
   },
   {
     name: 'Gesa Müller',
-    tasks: [],
+    tasks: ['Logo Design und Qualitätssicherung'],
```

Comment:
Es wäre gut, wenn du die einzelnen Aufgaben in einzelne Array-Elemente schreiben würdest, also:
```
tasks: ['Logo Design', 'Qualitätssicherung']
```